src/test/java/systemtests/AddressBookSystemTest.java

```diff
@@ -137,7 +137,7 @@ protected void executeCommand(String command) {
      * Displays all persons in the address book.
      */
```

Comment:
It would be good to change the comments from persons to healthworkers and etc.

Reply:
I think that since AddressBook related classes will be phased out eventually, it might not be top priority to refactor the comments relating to original AB4.